app.js

```diff
@@ -126,6 +126,12 @@ const render = () => {
 		document.getElementById('event-list').innerHTML += Mustache.render(TEMPLATE, event)
 	}
 
+	[...document.getElementsByClassName("event-header")].forEach(node => {
```

Comment:
Akú má toto výhodu oproti `document.getElementsByClassName("event-header").forEach(...)`?

Reply:
že `document.getElementsByClassName("event-header").forEach(...)` nefunguje lebo `getElementsByClassName` vracia `HTMLCollection` a ten nemá `forEach` funkciu. A teda toto viac preferujem ako klasický forcyklus.

Reply:
:thinking: to som nevedel. díky :)